test/integration/quic_http_integration_test.cc

```diff
@@ -1203,7 +1203,7 @@ TEST_P(QuicHttpIntegrationTest, DeferredLogging) {
   std::vector<std::string> metrics = absl::StrSplit(log, ",");
   ASSERT_EQ(metrics.size(), 21);
   EXPECT_EQ(/* PROTOCOL */ metrics.at(0), "HTTP/3");
-  EXPECT_GT(/* ROUNDTRIP_DURATION */ std::stoi(metrics.at(1)), 0);
```

Comment:
If this returns 0, we can't tell if ROUNDTRIP_DURATION is actually 0ms or not working at all.
Can you add some delay at the fake_upstream_ end?

Reply:
Thanks; used a large response size which should ensure RT > 0 instead.

Reply:
I'm not sure sending 1MB will guarantee a non-zero RTT. A rate faster than 1GB/s would be under 1ms, and on localhost on a fast server that seems possible. Can you do what @danzh2010 suggested and force some delay?